src/embedded.ts

```diff
@@ -69,27 +70,30 @@ export class RoomWidgetClient extends MatrixClient {
         super(opts);
 
         // Request capabilities for the functionality this client needs to support
-        this.capabilities.sendState?.forEach(({ eventType, stateKey }) =>
-            this.widgetApi.requestCapabilityToSendState(eventType, stateKey),
+        if (capabilities.sendState?.length || capabilities.receiveState?.length) {
+            widgetApi.requestCapabilityForRoomTimeline(roomId);
```

Comment:
Why is this necessary now and it wasn't before?

Reply:
Because the timeline capability is a requirement for accessing a specific room, even if that room is normally what you'd think of as "the current room". The widget API rejects any requests directed at a specific room if the widget doesn't have this capability.